File: src/main/java/com/jiaoay/rime/core/RimeCandidate.java
package com.jiaoay.rime.core;

/**
 * Rime候選項
 */
public class RimeCandidate {
    public String text;
    public String comment;

    public RimeCandidate(String text, String comment) {
        this.text = text;
        this.comment = comment;
    }

    public RimeCandidate() {
    }
}

File: src/main/java/com/jiaoay/rime/core/RimeCandidate.kt
package com.jiaoay.rime.core

/**
 * Rime候選項
 */
class RimeCandidate {
    @JvmField
    var text: String? = null
    @JvmField
    var comment: String? = null

    constructor(text: String?, comment: String?) {
        this.text = text
        this.comment = comment
    }

    constructor() {}
}